### PR TITLE
Doc fix

### PR DESCRIPTION
**Pull request by kazaamjt**

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -62,7 +62,7 @@ An Inmanta project bundles modules that contain configuration information. A pro
 than a directory with a project.yml file, which contains parameters such as the location to search for
 modules and where to find the server.
 
-Here we will get a project from github.
+You can retrieve the quickstart module like this:
```
Comment (arnaudsjs) [inline docs/quickstart.rst]:
```suggestion
You can retrieve the quickstart project like this:
```

File: docs/quickstart.rst [hunk 2]
```diff
@@ -157,7 +156,7 @@ Then compile the project and send it to the server:
 The first time you run this command may take a while, as all dependencies are downloaded.
 
 When the model is sent to the server, it will start deploying the configuration.
-To track progress, you can go to the `dashboard <http://127.0.0.1:8888>`_.
+To track progress, you can go to the `dashboard <http://127.0.0.1:8888>`_, select the `temp` project and then the `quickstart-env` environment.
```
Comment (arnaudsjs) [inline docs/quickstart.rst]:
```suggestion
To track progress, you can go to the `dashboard <http://127.0.0.1:8888>`_, select the `test` project and then the `quickstart-env` environment.
```

Comment (kazaamjt) [reply docs/quickstart.rst]: resolved

Comment (arnaudsjs) [reply docs/quickstart.rst]: It seems that this issue was not resolved in the source code.

**Pull request merged**